backend/CMakeLists.txt

```diff
@@ -16,3 +19,20 @@ add_executable(${PROJECT_NAME} ${SOURCES})
 target_link_libraries(${PROJECT_NAME} Crow::Crow SQLite::SQLite3)
 
 target_include_directories(${PROJECT_NAME} PRIVATE src)
+
+## tests
```

Comment:
**thought**: There might be a better way to separate these two things. I'll look into it.